pkg/daemon/discover/discover.go

```diff
@@ -398,11 +398,16 @@ func updateDeviceCM(context *clusterd.Context) error {
 func probeDevices(context *clusterd.Context) ([]sys.LocalDisk, error) {
 	devices := make([]sys.LocalDisk, 0)
 	localDevices, err := clusterd.DiscoverDevices(context.Executor)
-	logger.Infof("localdevices: %+v", localDevices)
 	if err != nil {
 		return devices, fmt.Errorf("failed initial hardware discovery. %+v", err)
 	}
 
+	var devicesList []string
+	for _, device := range localDevices {
+		devicesList = append(devicesList, fmt.Sprintf("%+v", device))
+	}
+	logger.Infof("localdevices: %q", strings.Join(devicesList, ", "))
```

Comment:
```suggestion
	logger.Infof("discovered localdevices: %q", strings.Join(devicesList, ", "))
```

What about a small function for this along with unit tests?

Reply:
@leseb So basically a function to generate a string from the `localDevices` (/`[]*sys.Disk`) var?

Reply:
Just get the loop you added into a new function that receives `localDevices`.

Reply:
@leseb There should a possibility to do unit testing on the `logger` though I'm not quiet sure it really "adds" having such a unit test.
What do you think?